transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformer/src/main/resources/js/typeMappingsSanitizer.js

```diff
@@ -281,7 +285,18 @@ function processBulkIndex(docBackfillPair, context) {
     return docBackfillPair;
 }
 
+// Helper to print nested maps
+function replacer(key, value) {
```

Comment:
What's up w/ this function?  It's called replacer and takes a key.  That doesn't match the comment or the code.